pkg/types/alibabacloud/platform.go

```diff
@@ -0,0 +1,13 @@
+package alibabacloud
+
+// Platform stores all the global configuration that all machinesets use.
+type Platform struct {
+	// Region specifies the Alibaba Cloud region where the cluster will be created.
+	Region            string `json:"region"`
+	ResourceGroupName string `json:"resourceGroupName"`
+}
+
+func (p *Platform) SetBaseDomain() error {
+	// pass
+	return nil
+}
```

Comment:
We should avoid adding code in one commit and then removing in another within the same PR. It would make it easier to review by rebasing/ammeding commits.

Reply:
OK